demo/admin/src/products/ManufacturersGrid.tsx

```diff
@@ -64,16 +64,15 @@ export function ManufacturersGrid() {
             width: 150,
             renderHeader: () => (
                 <div style={{ display: "flex", alignItems: "center" }}>
-                    <Typography fontWeight={400} fontSize={14}>
+                    <GridColumnHeaderTitle label="ID" columnWidth={150}>
                         {intl.formatMessage({ id: "manufacturers.id", defaultMessage: "ID" })}
-                    </Typography>
+                    </GridColumnHeaderTitle>
```

Comment:
`label` should be a formatted message and `children` should not be necessary, if the `label` is set.